site/fluxctl.md

```diff
@@ -188,13 +176,24 @@ Using an SSH key allows you to maintain control of the repository. You
 can revoke permission for `flux` to access the repository at any time
 by removing the deploy key.
 
-```sh
+```
```

Comment:
I think we may need to rethink about including this output in the docs (or automate it?) as it is likely to change over time.

Reply:
I was thinking the same thing.  In the past I have used https://github.com/danger/danger which basically would allow us to write code that enforces that when the help text changes that the documentation will/should also change.  That's the best option I know of (and, there are ways in which it isn't great) that meets the minimum requirement of curbing some of that stuff getting out of date.

Another option would be to just have a quarterly health-and-wellness checkup list that would include updating documentation of this sort.  Just an idea.